Makefile.sub

```diff
@@ -91,11 +96,8 @@ HLS_TEST_BASE_FILES = \
 	base/test_cpu \
 
 .PHONY: hls-test-base
-hls-test-base: HLS_TEST_FUNCTION=run
-hls-test-base: $(patsubst %, $(HLS_TEST_BUILD)/%.hls, $(HLS_TEST_BASE_FILES))
 hls-test-base: $(patsubst %, hls-test-run/%.hls, $(HLS_TEST_BASE_FILES))
 
-
```

Comment:
Should you not also remove the `HLS_TEST_FUNCTION=kernel` line for the polybench benchmarks?

Reply:
And also that line: `hls-test-polybench: $(patsubst %, $(HLS_TEST_BUILD)/%.hls, $(HLS_TEST_POLYBENCH_FILES))`

Reply:
Sorry, I was too hasty in the previous commit.
Have removed them now.

Reply:
What about my second comment? Should this line not also be removed like for all the other benchmark suites?

Reply:
You did not really perform the changes asked above, just commented. Did you forget to push?

Reply:
Yes, forgot to push the change.